src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,165 @@
+        String expected = "Password can't be less than 6 symbols";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_lengthOfPasswordIs5_NotOk() {
+        String password = "small";
+        user.setPassword(password);
+        Exception exception =
+                assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        String expected = "Password can't be less than 6 symbols";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_NullLogin_NotOk() {
+        user.setLogin(null);
+        Exception exception =
+                assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        String expected = "Login can't be null";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_emptyLogin_NotOk() {
+        user.setLogin("");
+        Exception exception =
+                assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        String expected = "Login can't be empty";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_lengthOfLoginIs3_NotOk() {
+        String login = "asd";
+        user.setLogin(login);
+        Exception exception =
+                assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        String expected = "Login can't be less than 6 symbols";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_lengthOfLoginIs5_NotOk() {
+        String login = "small";
+        user.setLogin(login);
+        Exception exception =
+                assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        String expected = "Login can't be less than 6 symbols";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_nullAge_NotOk() {
+        user.setAge(null);
+        Exception exception =
+                assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        String expected = "Age can't be null";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(expected));
+    }
+
+    @Test
+    void register_userWithExistingLogin_NotOk() {
+        User newUser = new User();
+        newUser.setLogin("someLogin");
+        newUser.setPassword("goodPassword");
+        newUser.setAge(22);
+        registrationService.register(newUser);
+        assertThrows(RegistrationException.class, () -> registrationService.register(newUser));
```

Comment:
It would be beneficial to verify the exception message here as well, to ensure that the correct reason for the exception is being thrown, similar to other tests.